File: src/main/java/com/sfl/tms/api/common/annotations/ValidateActionRequest.java
package com.sfl.tms.api.common.annotations;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * User: Vazgen Danielyan
 * Date: 12/25/18
 * Time: 1:00 PM
 */
@Retention(RetentionPolicy.RUNTIME)
@Target({ElementType.PARAMETER, ElementType.METHOD, ElementType.FIELD})
public @interface ValidateActionRequest {
}

File: src/main/kotlin/com/sfl/qup/tms/api/common/annotations/ValidateActionRequest.kt
package com.sfl.qup.tms.api.common.annotations

/**
 * User: Vazgen Danielyan
 * Company: SFL LLC
 * Date: 1/25/16
 * Time: 11:20 PM
 */
@Retention(AnnotationRetention.RUNTIME)
@Target(AnnotationTarget.VALUE_PARAMETER, AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER, AnnotationTarget.FIELD)
annotation class ValidateActionRequest
